### ISSUE TITLE
Unable to remove relationship between Account and Archive Email

### ISSUE THREAD

**Issue by samus-aran** [salesagility/SuiteCRM#634]
Navigate to an Accounts record > History sub panel > any Archived Email record:

Click on the record's subpanel 'Remove' Button via the 'Edit' button.

A Popup will appear to confirm you wish to delete the relationship and the record will not be deleted - click OK.

Once clicked the Subpanel will re-sort (to what reason don't know) but the Archive Email is still present in the list.

**Comment by paulgodley** [comment]
This is also the case for "In Draft" and "Sent" email records in the History sub-panel.  The remove function appears to work for the other categories.

**Comment by paulgodley** [comment]
When the email is removed from the history, the deleted flag is set to 0 in emails_beans.  This is probably not being checked when the emails are being loaded for the account.

**Comment by Mausino** [comment]
Same issue is related to #498

**Comment by JimMackin** [comment]
We may need to discuss how this is handled. The history subpanel can show emails directly assigned to the account, assigned to a contact related to the account or matched by email address only.

It's not clear how this should behave if the email is assigned in the various different ways above and is then removed.  Any thoughts @mattlorimer?

**Comment by mattlorimer** [comment]
In this instance where its is showing an email by email address, not a direct relationship or by email address and a direct relationship then I see the only option is for the option to remove it,  not to be shown. As the only other option, apart from configuring your instance not to show emails by email address or by contact emails address, is to have an option to completely delete the email, or to remove the related email address from the email, which may not be practical

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Unable to remove relationship between Account and Archive Email](https://trello.com/c/U4V7NXmu/138-unable-to-remove-relationship-between-account-and-archive-email)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
